src/getPropertyByPath.ts

```diff
@@ -5,7 +5,7 @@
 // understood in array paths.
 function getPropertyByPath(
   source: { [key: string]: unknown },
-  path: string | Array<string>,
+  path: string | string[],
```

Comment:
I prefer the generic syntax, `Array<string>`, because it stands out more clearly.

Reply:
This comes from their recommended preset. Should I override?

Reply:
Fixed via [`'@typescript-eslint/array-type': ['error', 'generic']`](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/array-type.md) - https://github.com/davidtheclark/cosmiconfig/pull/202/commits/a9e53cce3df0dff34e43eea2b18e57be302c838a